src/main/kotlin/com/server/dpmcore/security/oauth/handler/CustomAuthenticationSuccessHandler.kt

```
@@ -20,14 +21,31 @@ class CustomAuthenticationSuccessHandler(
         }
     }
 
+    private fun getRequestDomain(request: HttpServletRequest): String {
+        val raw =
+            request.getHeader("Origin")
+                ?: request.getHeader("Referer")
+                ?: request.requestURL.toString()
+
+        return try {
+            URI(raw).host
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)